src/test/java/com/yelp/nrtsearch/server/grpc/LuceneServerTest.java

```diff
@@ -432,7 +432,7 @@ public void testStats() throws IOException, InterruptedException {
     assertEquals(0, statsResponse.getOrd());
     assertEquals(2, statsResponse.getCurrentSearcher().getNumDocs());
     assertEquals(1, statsResponse.getCurrentSearcher().getNumSegments());
-    assertEquals(6610, statsResponse.getDirSize(), 1000);
+    assertEquals(6610, statsResponse.getDirSize(), 1500);
```

Comment:
any reason about this change? Did you change anything related?

Reply:
My changes did end up causing this issue. However, this assert might end up being flaky, because it's calculating a directory size which can vary based on a number of factors.